packages/frontend/src/components/UserSettings/AppearanceSettingsPanel/DeletePaletteModal.tsx

```diff
@@ -26,63 +21,25 @@ export const DeletePaletteModal: FC<DeletePaletteModalProps> = ({
     onClose,
     onConfirm,
 }) => (
-    <Modal
-        radius="sm"
+    <MantineModal
         opened={opened}
         onClose={onClose}
-        title={
-            <Group>
-                <Paper p="xs" withBorder radius="sm">
-                    <MantineIcon icon={IconTrash} size="sm" color="red" />
-                </Paper>
-                <Text color="ldDark.7" fw={500} fz="md">
-                    Delete "{palette.name}" Palette
-                </Text>
-            </Group>
-        }
-        styles={(theme) => ({
-            header: {
-                borderBottom: `1px solid ${theme.colors.ldGray[2]}`,
-            },
-            body: {
-                padding: 0,
-            },
-        })}
+        title={`Delete "${palette.name}" Palette`}
+        icon={IconTrash}
```

Comment:
in some places we use `iconTrash` and then in others `IconAlert` for the same delete action.  
I think it would be nice to unify all of these with a generic variant `delete` that adds the same icon to the modal and sets the action to red. We can even include a generic message like "are you sure you want to delete ${model}"   
We could do the same for `create` , use a `IconPlus` for example and primary color for the action.  
wdyt?